.github/workflows/create-portable-zip.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+import os
+import sys
+import zipfile
+
+import gha
+
+if len(sys.argv) != 3:
+    gha.print_error('Usage: create-portable-zip.py <output_path> <release|debug>')
+    sys.exit(1)
+else:
+    output_path = sys.argv[1]
+    configuration = sys.argv[2].lower()
+
+with zipfile.ZipFile(output_path, 'x', zipfile.ZIP_DEFLATED, compresslevel=9) as output:
```

Comment:
I said during our meeting that compresslevel here was to override the default of not compressing at all. That wasn't right, that is true for the previous parameter (specified as ZIP_DEFLATED) here.

I specified it as 9 mainly because the Python documentation didn't specify what the default was otherwise and this step is so fast that speed wasn't a concern.

I dug around in the Python source code and found that omitting it passes `Z_DEFAULT_COMPRESSION` to zlib which [documents that as level 6](http://www.zlib.net/manual.html).

--------

I was also curious about the savings so I did a little experiment and it's actually a bit bigger lol.

* Bonsai.zip as published on GitHub: 1.34 MB
* Bonsai.zip compressed by Python at level 9: 1.38 MB
* Bonsai.zip compressed by 7zip at level 9: 1.32 MB